dockertest/images.py

```diff
@@ -430,6 +430,17 @@ def default_image(self):  # pylint: disable=R0201
         # DICLS may have overriden this method
         return DockerImage.full_name_from_defaults(defaults)
 
+    def default_image_exist_check(self):
```

Comment:
It may depend on how we using it
If name it as ``pull``, i'd like to use it like
```
if image not exist:
           pull(self)
container.execute()
```
otherwise, it will looks like
```
images.check
container.execute()
```
i'd rather use the second one since it will appear every case basically, 
just save the duplicate thing, even though the first one looks like better.
Any suggestion?